cedar-drt/tests/integration_tests.rs

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Integration test that runs the handwritten test cases from
+//! `cedar-integration-tests` on the definitional implementation.
+
+use cedar_policy::integration_testing::{
+    perform_integration_test_from_json_custom, resolve_integration_test_path, CustomCedarImpl,
+};
+
+use cedar_drt::*;
+use std::path::Path;
+
+// Note: Some of the following code is based on cedar-policy/tests/corpus_tests.rs, but
+// there probably isn't enough commonality left to be worth trying to share the
+// code.
```

Comment:
in fact, I think we can just remove this comment at this point -- almost all the sharable code is shared